publications.md

```diff
@@ -5,7 +5,7 @@ permalink: /publications/
 ---
 {%- if site.data.publications.size > 0 -%}
 {%- for publication in site.data.publications -%}
-# [{{ publication.title }}]({{ publication.link }})
+## [{{ publication.title }}]({{ publication.link }})
```

Comment:
`h2`?

Reply:
Every page now begins with <h1>Aerosec:...</h1>, so I thought this looked better?

Reply:
Oh, I see now. I missed that in the changed `default.html`.